docs/faq.md

```diff
@@ -3,6 +3,9 @@
 ## Why is my version not incrementing?
 GitVersion calculates the semantic version, this will only change once per *release*. Read more at [version increments](more-info/version-increments.md)
 
+## I'm using Octopus deploy
```

Comment:
Wouldn't say I had thought a lot about it. Not sure how to phrase, just want to make the octopus deploy stuff more discoverable